<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igVector</class>
 <widget class="QWidget" name="igVector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>矢量可视化</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>头部直径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="headRSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>头部长度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="headLSlider_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>尾部直径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="tailRSlider_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>尾部长度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="tailLSlider_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>数据来源</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="headRlineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="headLlineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tailRlineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tailLlineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
